### ISSUE TITLE
How can i change the z-index of direction arrow buttons?

### ISSUE THREAD

**Issue by nicomontanari98** [ascoders/react-native-image-viewer#203]
![screen](https://user-images.githubusercontent.com/15747776/41922423-589e8ab0-7965-11e8-9354-c9832537f6c5.gif)

I tried to increase z-index and elevation but nothing has changed.

**Comment by cyperus** [comment]
I think you need to modify  this file `src/image-viewer.style.ts  ` at line  91
```
arrowLeftContainer: {
      position: 'absolute',
      top: 0,
      bottom: 0,
      left: 0,
      justifyContent: 'center',
    zIndex:99 //add here
  },
  arrowRightContainer: {
      position: 'absolute',
      top: 0,
      bottom: 0,
      right: 0,
      justifyContent: 'center',
     zIndex:99 //add here
    }
```

**Comment by ascoders** [comment]
Fixed in `2.2.8`
